libusb/hid.c

```diff
@@ -1247,6 +1257,15 @@ void HID_API_EXPORT hid_close(hid_device *dev)
 	/* release the interface */
 	libusb_release_interface(dev->device_handle, dev->interface);
 
+	/* reacttach the kernel driver if it was detached */
+#ifdef DETACH_KERNEL_DRIVER
+	if (dev->is_driver_detached){
+		int res = libusb_attach_kernel_driver(dev->device_handle, dev->interface);
+		if (res<0)
```

Comment:
```suggestion
		if (res < 0)
```

Reply:
All changes have been implemented in the commit 8d4cc17.